### PR TITLE
Create ML_8.6_Transformers (D) .md

### PR DESCRIPTION
**Pull request by shrestharushika**
<hr>

## Description 📜

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. 
Fixes #issue_number_here 

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [ ] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [ ] JavaScript_#Domain
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [ ] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [ ] I have performed a self-review of my own code or work.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] My changes generates no new warnings.
- [ ] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

File: Machine_Learning/ML_For_Sequential_Data_&_Visual_Data/ML_8.6_Transformers (D) .md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+**Rushika Shrestha**                                                                  
+**ML Batch-1
+Domain- Machine Learning**
+
```
Comment (Isha307) [inline Machine_Learning/ML_For_Sequential_Data_&_Visual_Data/ML_8.6_Transformers (D) .md:4]: Remove your name and batch number.

**Review by Isha307 — CHANGES_REQUESTED** [review]
And follow this https://github.com/girlscript/winter-of-contributing/blob/Machine_Learning/Machine_Learning/Demo/ML_%23_%23_Documentation%20Template_(D).md and also try to add some more content with clear explanation.

**Comment by abhisheks008** [comment]
@shrestharushika Do the needful changes.

**Comment by abhisheks008** [comment]
@shrestharushika You have deleted the title in the markdown file. Add the title.

**Review by Isha307 — CHANGES_REQUESTED** [review]
Please change 8.6 to 8_6 in the file name.

**Review by Isha307 — APPROVED** [review]

**Pull request merged**